XRTK-Core/Packages/com.xrtk.core/Inspectors/MixedRealityShaderGUI.cs

```diff
@@ -0,0 +1,408 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
Why is it acting like this is a new file?

I'd like to see the diff 😟

Reply:
Because they split up the Original Mixed Reality shader in to two parts, 1. common, 2. standard.
Then used the common to build up the Wireframe shader.

Reply:
I'd prefer we be able to see the diff.